setup.cfg

```diff
@@ -47,6 +47,6 @@ skip=
     edxval/wsgi.py
 
 [tool:pytest]
-DJANGO_SETTINGS_MODULE = edxval.settings.base
+DJANGO_SETTINGS_MODULE = edxval.settings.test
```

Comment:
How are we providing django settings settings while running pact with `pytest`?

Reply:
like we did with LMS, by exporting the settings module env variable before executing the verify call.